recipe/run_test.py

```diff
@@ -97,5 +97,3 @@
 
 import ssl
 print('OPENSSL_VERSION:', ssl.OPENSSL_VERSION)
```

Comment:
Why not:
```suggestion
print('OPENSSL_VERSION:', ssl.OPENSSL_VERSION)
CONDA_OPENSSL_VERSION = os.getenv("openssl")
assert CONDA_OPENSSL_VERSION in ssl.OPENSSL_VERSION
```

(Inspired by similar changes by Isuru around `tk` for the 3.10 PR)